getting-started.md

```diff
@@ -0,0 +1,64 @@
+# Getting Started: Docker Runbook
+
```

Comment:
this file should be moved to the .dev folder so its not distributed with the plugin